README.md

```diff
@@ -85,6 +88,42 @@ steps:
       approve-only: true
 ```
 
+### Trigger action from workflow_dispatch event
+
+If you need to trigger this action manually, you can use the [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) event. A use case might be that your CI runs on a seperate provider, so you would like to run this action as a result of a successful CI run.
+
+When using the `workflow_dispatch` approach, you will need to send the PR number as part of the input for this action:
+
+```yml
+name: automerge
+
+on: 
+  workflow_dispatch:
+    inputs:
+      pr:
```

Comment:
for consistency with the name of the action parameter, let's call this `pr-number`

```suggestion
      pr-number:
```

Reply:
Good idea!